cmd/destroy/destroy.go

```diff
@@ -65,6 +65,7 @@ type Options struct {
 	ManifestPath        string
 	Name                string
 	Variables           []string
+	Manifest            *model.Manifest
```

Comment:
we need to propagate the Manifest to be able to get the image name in the error message

Reply:
Let's try to not use the options to store the manifest. Maybe we can store it in the `remoteDestroyCommand` struct adding it [here](https://github.com/okteto/okteto/pull/3494/files#diff-a1fde665f65ea0f657c3cc6aff6529b33ec5b8c7cd15596b7418a3ea5d8669d3R259). This will reduce a lot the complexity of the changes.

We can change `newRemoteDestroyer` signature to receive a manifest instead of the destroyImage name and that will should be simpler. WDYT?

Reply:
I have used the same approach for the `build` and did the change fairly quickly so I didn't stop too much thinking about other ways to store, so thanks, I will update it now

Reply:
@jLopezbarb I've started making the change, but realized that the error message is composed [here](https://github.com/okteto/okteto/blob/master/pkg/cmd/build/build.go#L177-L184) and if `buildOptions.Manifest` is nil, it won't work - that's because `build` already had the Manifest in the options. 

I will think about your approach, but I foresee other changes needed to make it work

Reply:
I looked with fresh eyes at the code after lunch and your suggestion makes 100% sense. I've updated the code now. Thanks @jLopezbarb!